pkg/rsyslog/Dockerfile.in

```diff
@@ -0,0 +1,20 @@
+# hadolint ignore=DL3006
+FROM GPTTOOLS_TAG as gpttools
+
+FROM alpine:3.8
+WORKDIR /
+COPY rsyslog@lists.adiscon.com-5a55e598.rsa.pub /etc/apk/keys/rsyslog@lists.adiscon.com-5a55e598.rsa.pub
+# hadolint ignore=DL3018
```

Comment:
What's the purpose of this pubkey?

Reply:
Without adding this public key, I see Docker build failing with the following error.
WARNING: Ignoring http://alpine.rsyslog.com/3.7/stable/x86_64/APKINDEX.tar.gz: UNTRUSTED signature
ERROR: unsatisfiable constraints:

I copied this Docker build file from https://github.com/rsyslog/rsyslog-docker/blob/master/appliance/alpine/Dockerfile

Reply:
Hey @gkodali-zededa -- it is usually a much better choice to always use upstream Alpine packages instead of people maintaining these packages on the side. So. Here's what you need to do here: drop this key and move to Alpine 4.10.3 as a base for your Dockerfile. Then your Dockerfile will look like this:

```
FROM alpine:3.10.3
WORKDIR /
RUN apk add --no-cache rsyslog=XXX rsyslog-mmjsonparse=YYY
COPY rsyslog.conf /etc/rsyslog.conf
COPY init.sh /init.sh

ENTRYPOINT []
CMD ["/init.sh"]
```

And yes -- you'll have to determine values for XXX and YYY

Reply:
Will do.